src/main/java/Main.java

```diff
@@ -0,0 +1,110 @@
+import java.util.Scanner;
+
+public class Main {
+    public static final String DIVIDER = "--------------------------";
+    public static final String NL = System.lineSeparator(); // New Line
+
+    private static int count = 0;
+    private static final Task[] list = new Task[100];
+
+    public static void checkInput(String msg) {
```

Comment:
Avoid lengthy methods. Keep the length of each method to about 30 lines of code.